lua/render-markdown/colors.lua

```diff
@@ -80,7 +80,12 @@ M.inverse = function(highlight)
     local name = string.format('%s_Inverse_%s', M.prefix, highlight)
     if not vim.tbl_contains(cache.highlights, name) then
         local hl = M.get_hl(highlight)
-        vim.api.nvim_set_hl(0, name, { fg = hl.bg, bg = hl.fg })
+        vim.api.nvim_set_hl(0, name, {
```

Comment:
Thanks for the PR! Should this be applied to the `combine` method as well?

Reply:
No problem, thanks for the plugin :slightly_smiling_face: 
I think so, I just pushed an update that should address that. I had that part disabled so I didn't notice.

Reply:
Makes sense, thanks for the update!